lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -151,12 +151,16 @@ function paraneue_dosomething_preprocess_page(&$vars) {
       'title' => drupal_get_title(),
     );
     // If there's a subtitle field on this node, send it to the header
-    if (isset($vars['node']->field_subtitle[LANGUAGE_NONE][0]['safe_value'])) {
-      $header_vars['subtitle'] = $vars['node']->field_subtitle[LANGUAGE_NONE][0]['safe_value'];
+    global $user;
+    $language = $user->language;
```

Comment:
Not sure off hand, but does this run when it's an anonymous user? Cause this will result in `undefined` errors in the **dblog** if this gets loaded at whatever point by user that's not logged in...